File: C#/C# Advanced/Methods/06_FirstLargerThanNeighbours/FirstLargerThanNeighbours.cs
using System;
using System.Linq;

class FirstLargerThanNeighbours
{
    static void Main()
    {
        int arrSize = int.Parse(Console.ReadLine());
        int[] numbers = new int[arrSize];

        numbers = Console.ReadLine().Split(new char[] { ' ' },
            StringSplitOptions.RemoveEmptyEntries).Select(int.Parse).ToArray();

        Console.WriteLine(GetIndexOfFirstLargerThanNeighbours(numbers));
    }

    static int GetIndexOfFirstLargerThanNeighbours(int[] arr)
    {
        for (int i = 1; i < arr.Length - 1; i++)
        {
            if (arr[i] > arr[i + 1] && arr[i] > arr[i - 1])
            {
                return i;
            }
        }

        return -1;
    }
}

File: Javascript/Javascript Fundamentals/Functions/first-larger-than-neighbours.js
function solve(args) {
    var n = +args[0],
    arr = args[1].split(' ');

    console.log(FirstLarger(arr));

    function FirstLarger(array) {
        var i;
        for(i = 0; i < array.length - 1; i += 1) {
            if(+array[i] > +array[i + 1]){
                return i;
            }
        }
        return -1;
    }
}